package.json

```diff
@@ -26,29 +28,50 @@
       "latest"
     ],
     "plugins": [
-      "transform-class-properties"
+      "transform-class-properties",
+      "transform-object-rest-spread",
+      [
+        "module-resolver",
+        {
+          "root": [
+            "./src"
+          ]
+        }
+      ]
```

Comment:
Adds package module resolution so you can use `package/components/Icon` instead of `../../../../package/components/Icon` in both the app and package.